content/en/news/2019/announcing-1.1-eol-final/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.1 has ended
+description: Istio 1.1 end of life announcement.
+publishdate: 2019-08-18
+attribution: The Istio Team
+aliases:
+    - /blog/2019/announcing-1.1-eol-final
```

Comment:
No aliases for new files, they're only used when moving old files.

Reply:
done